services/order.py

```diff
@@ -0,0 +1,42 @@
+from django.db import transaction
+from django.db.models import QuerySet
+from db.models import Ticket, Order, get_user_model, MovieSession
+
+
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: str = None) -> None:
+
+    with transaction.atomic():
+
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+        tickets_instances = []
+        if date:
+            order.created_at = date
+        order.save()
```

Comment:
You are calling the `order.save()` method inside the `if date:` block. According to the checklist, you should avoid calling `save()` multiple times if possible. Consider moving the `order.save()` call outside the `if date:` block to ensure it is called only once.